src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,6 @@
+$font-color: #292929;
+$font-color-form: #7e7e83;
+$pargraph-font-family: "Space Mono", sans-serif;
```

Comment:
There's a typo in the variable name $pargraph-font-family. It should be $paragraph-font-family.